src/test/java/uk/gov/hmcts/reform/wataskmanagementapi/cft/query/TaskResourceSpecificationTest.java

```diff
@@ -296,47 +299,53 @@ void shouldBuildSingleTaskQueryForCompletableNegated() {
 
         searchTaskRequest = new SearchTaskRequest(List.of(
-            new SearchParameter(STATE, SearchOperator.IN, singletonList("ASSIGNED"))
+            new SearchParameterList(STATE, SearchOperator.IN, singletonList("ASSIGNED"))
         ));
         final SearchTaskRequestScenario state =
             SearchTaskRequestScenario.builder().searchTaskRequest(searchTaskRequest).build();
 
         searchTaskRequest = new SearchTaskRequest(List.of(
-            new SearchParameter(LOCATION, SearchOperator.IN, singletonList("location"))
+            new SearchParameterList(LOCATION, SearchOperator.IN, singletonList("location"))
         ));
         final SearchTaskRequestScenario location =
             SearchTaskRequestScenario.builder().searchTaskRequest(searchTaskRequest).build();
 
         searchTaskRequest = new SearchTaskRequest(List.of(
-            new SearchParameter(CASE_ID, SearchOperator.IN, singletonList("caseId"))
+            new SearchParameterList(CASE_ID, SearchOperator.IN, singletonList("caseId"))
         ));
         final SearchTaskRequestScenario caseId =
             SearchTaskRequestScenario.builder().searchTaskRequest(searchTaskRequest).build();
 
         searchTaskRequest = new SearchTaskRequest(List.of(
-            new SearchParameter(USER, SearchOperator.IN, singletonList("testUser"))
+            new SearchParameterList(USER, SearchOperator.IN, singletonList("testUser"))
         ));
         final SearchTaskRequestScenario user =
             SearchTaskRequestScenario.builder().searchTaskRequest(searchTaskRequest).build();
 
         searchTaskRequest = new SearchTaskRequest(List.of(
-            new SearchParameter(WORK_TYPE, SearchOperator.IN, singletonList("routine_work"))
+            new SearchParameterList(WORK_TYPE, SearchOperator.IN, singletonList("routine_work"))
         ));
         final SearchTaskRequestScenario workType =
             SearchTaskRequestScenario.builder().searchTaskRequest(searchTaskRequest).build();
 
         searchTaskRequest = new SearchTaskRequest(List.of(
-            new SearchParameter(ROLE_CATEGORY, SearchOperator.IN, singletonList("LEGAL_OPERATIONS"))
+            new SearchParameterList(ROLE_CATEGORY, SearchOperator.IN, singletonList("LEGAL_OPERATIONS"))
         ));
         final SearchTaskRequestScenario roleCtg =
             SearchTaskRequestScenario.builder().searchTaskRequest(searchTaskRequest).build();
 
+        //searchTaskRequest = new SearchTaskRequest(List.of(
+        //    new SearchParameterBoolean(AVAILABLE_TASKS_ONLY, SearchOperator.BOOLEAN, true)
+        //));
+        //final SearchTaskRequestScenario availableTaskOnly =
+        //    SearchTaskRequestScenario.builder().searchTaskRequest(searchTaskRequest).build();
+        //return Stream.of(jurisdiction, state, location, caseId, user, workType, roleCtg, availableTaskOnly);
         return Stream.of(jurisdiction, state, location, caseId, user, workType, roleCtg);
```

Comment:
you have reverted my changes to fix this test. Can you compare this file with the my revision where I added expected count to the scenario